decidim-design/config/locales/en.yml

```
@@ -85,6 +85,14 @@ en:
           section_p_tablet: Tablet grid system is composed of 8 flexible columns with a gutter between columns of 16px and left and right margins of 24px
           tablet: Tablet
           title: Layout
+        typography:
+          description: These guidelines summarize how use typography at Decidim, and serve as guardrails for designers to design freely with best typographic practices in mind.
+          readability_description_1: Size, Layout cols, Characters per line
+          readability_description_2: Layout cols
+          readability_description_3: Characters per line
```

Let's add a namespace here and give them meaningful names. Also in the first value you're repeating the 2nd and the 3rd
```suggestion
          headings:
            characters: Characters per line
            layout: Layout cols
            size: Size
```